jupyterhub_config.py

```diff
@@ -1,13 +1,110 @@
 from jupyterhub.spawner import SimpleLocalProcessSpawner
 
+from jhub_apps import theme_template_paths
 from jhub_apps.configuration import install_jhub_apps
 
 c = get_config()  # noqa
 
 from jupyterhub.auth import DummyAuthenticator  # noqa: E402
 
+c.Authenticator.admin_users = {"admin"}
 c.JupyterHub.authenticator_class = DummyAuthenticator
 c.JupyterHub.log_level = 10
 
-c.JupyterHub.bind_url = "http://127.0.0.1:8000"
+hub_url = "http://127.0.0.1:8000"
+
+c.JupyterHub.bind_url = hub_url
 c = install_jhub_apps(c, spawner_to_subclass=SimpleLocalProcessSpawner)
+
+c.JupyterHub.default_url = "/hub/home"
+
+c.JupyterHub.template_paths = theme_template_paths
+c.JupyterHub.services.extend(
+    [
+        {
+            "name": "JuypterLab",
+            "url": hub_url,
+            "display": True,
+            "info": {
+                "name": "JupyterLab",
+                "url": "/user/[USER]/lab",
+                "external": True,
+            },
+            "oauth_no_confirm": True,
+        },
```

Comment:
Currently the Service buttons are populated with default JupyterHub services. These are added to provide a template for adding more as needed